### PR TITLE
auction: add views to the `ActionView` definition

### PR DESCRIPTION
**Pull request by erwanor**
## Describe your changes

This PR adds `ActionDutchAuctionScheduleView` and `ActionDutchAuctionWithdrawView` to the `ActionView` message, it also replaces the `IsAction` perspective impls with concrete implementation.

Populating a view for the withdraw action without extending the `TxP` structure has proven squirrelly, and it will require a follow-up, although the data is "here" in the view server so I don't think it will be too complicated.

## Issue ticket number and link

Part of #4219 

## Checklist before requesting a review

- [x] If this code contains consensus-breaking changes, I have added the "consensus-breaking" label. Otherwise, I declare my belief that there are not consensus-breaking changes, for the following reason:

  > Breaking changes are made to `ActionView`: the `action_dutch_auction_schedule` and `action_dutch_auction_withdraw` fields have had their types changes from `ActionDutchAuctionSchedule` and `ActionDutchAuctionWithdraw` to `ActionDutchAuctionScheduleView` and `ActionDutchAuctionWithdrawView`, respectively.

### CODE REVIEW

File: proto/penumbra/penumbra/core/transaction/v1/transaction.proto [hunk 1]
```diff
@@ -223,9 +223,9 @@ message ActionView {
     component.governance.v1.CommunityPoolOutput community_pool_output = 51;
     component.governance.v1.CommunityPoolDeposit community_pool_deposit = 52;
     // Dutch auctions
-    component.auction.v1alpha1.ActionDutchAuctionSchedule action_dutch_auction_schedule = 53;
+    component.auction.v1alpha1.ActionDutchAuctionScheduleView action_dutch_auction_schedule = 53;
     component.auction.v1alpha1.ActionDutchAuctionEnd action_dutch_auction_end = 54;
-    component.auction.v1alpha1.ActionDutchAuctionWithdraw action_dutch_auction_withdraw = 55;
+    component.auction.v1alpha1.ActionDutchAuctionWithdrawView action_dutch_auction_withdraw = 55;
```
Comment (erwanor) [inline proto/penumbra/penumbra/core/transaction/v1/transaction.proto:228]: @jessepinho oversight on our end, but this should fix it.

**Comment by conorsch** [comment]
Does this break protos that were released, or just what's on main?

**Review by conorsch — APPROVED** [review]

**Comment by erwanor** [comment]
we publish to the BSR on main right? the blast damage should only be a branch in the web repo though

**Comment by cratelyn** [comment]
checking my understanding: this is _not_ a `consensus-breaking` change because it only performs breaking changes to `ActionView`, rather than `Action`.

**Review by cratelyn — APPROVED** [review]
this looks good to me! :hammer:

**Comment by erwanor** [comment]
@cratelyn that's right, however this would be a pretty bad client breaking change

**Comment by erwanor** [comment]
Let's break some protos, shall we 😎

**Pull request merged**